packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -489,6 +499,10 @@ export class Function extends FunctionBase {
       managedPolicies.push(iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSLambdaVPCAccessExecutionRole'));
     }
 
+    if (props.filesystems) {
+      managedPolicies.push(iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonElasticFileSystemClientFullAccess'));
```

Comment:
I would expect this to be part of the FileSystemConfig because this is EFS specific, no?

Reply:
working on it.

Reply:
OK. fixed.